### PR TITLE
deprecate jim : moved to MIRTjim.jim

### PR DESCRIPTION
**Pull request by JeffFessler**
Towards eventually eliminating Plots dependence from MIRT.

### CODE REVIEW

File: src/plot/jim.jl [hunk 1]
```diff
@@ -6,257 +6,9 @@ jiffy image display
...
-	if haskey(jim_def, test)
-		return jim_def[test]
-	end
-
-	throw("symbol $test")
-end
+@deprecate jim(a ; kwargs...) MIRTjim.jim(a ; kwargs...)
+@deprecate jim(a, b ; kwargs...) MIRTjim.jim(a, b ; kwargs...)
+@deprecate jim(a, b, c ; kwargs...) MIRTjim.jim(a, b, c ; kwargs...)
+@deprecate jim(a, b, c, d ; kwargs...) MIRTjim.jim(a, b, c, d ; kwargs...)
```
Comment (JeffFessler) [inline src/plot/jim.jl:14]: This repetition is a little bit ugly, but oh well.

Comment (johnnychen94) [inline src/plot/jim.jl:14]:
```suggestion
@deprecate jim(args... ; kwargs...) MIRTjim.jim(args...; kwargs...)
```

Comment (JeffFessler) [reply src/plot/jim.jl]: oops, after reading your other reply i was making a commit with that same change at the same time so now github won't let me accept your suggestion.   sorry!

**Comment by JeffFessler** [comment]
@johnnychen94 the documenter build is failing
https://github.com/JeffFessler/MIRT.jl/pull/96/checks?check_run_id=2649766360
apparently because the pkg server used by the host that runs documenter is *still* outdated w.r.t. MIRTjim 0.6.0, cf
https://github.com/JuliaRegistries/General/issues/16777
The other CI builds run fine, which is mysterious to me.
Do you think that the `ENV["JULIA_PKG_SERVER"] = ""` idea could be added to Documentation.yml somewhere as a workaround?

**Comment by johnnychen94** [comment]
> Do you think that the ENV["JULIA_PKG_SERVER"] = "" idea could be added to Documentation.yml somewhere as a workaround?

Of course, PR in #97

**Pull request merged**